### PR TITLE
Use postgraphile as express middleware

### PR DESCRIPTION
**Pull request by rmeena840**
Use postgraphile as express middleware
Task: https://www.pivotaltracker.com/story/show/182307046

### CODE REVIEW

File: backend/server.ts [hunk 1]
```diff
@@ -1,19 +1,23 @@
...
-	"/api",
-	graphqlHTTP({
-		schema: schema,
+	postgraphile(dbUrl, "public", {
+		ignoreRBAC: true,
+		extendedErrors: ["errcode", "detail", "hint"],
 		graphiql: true,
+		enhanceGraphiql: true,
+		graphqlRoute: "/api",
+		graphiqlRoute: "/graphiql",
```
Comment (Megha-Dev-19) [inline backend/server.ts:17]: can you please explain me the use of parameters or link a documentation?

**Review by Megha-Dev-19 — COMMENTED** [review]
lgtm, but I'm not much aware about postgraphile.

File: backend/server.ts (hunk 1 above)
Comment (rmeena840) [reply backend/server.ts:17]: For more info: https://www.graphile.org/postgraphile/usage-library/#http-middleware

**Comment by rmeena840** [comment]
@robert-zaremba Please review the PR.

File: backend/psql/config.ts [hunk 2]
```diff
@@ -1,5 +1,9 @@
-export const POSTGRES_UNAME = process.env.POSTGRES_UNAME || "algorand";
-export const POSTGRES_DBNAME = process.env.POSTGRES_DBNAME || "pgdb";
-export const POSTGRES_PASS = process.env.POSTGRES_PASS || "indexer";
-export const POSTGRES_HOST = process.env.POSTGRES_HOST || "localhost";
-export const POSTGRES_PORT = process.env.POSTGRES_PORT || 5432;
+// PostgreSQL parameters to pass to the URL origin
```
Comment (vuvoth) [inline backend/psql/config.ts:1]: You missing load env from file `require('dotenv').config()`.

File: backend/server.ts [hunk 3]
```diff
@@ -1,19 +1,23 @@
...
 app.use(
-	"/api",
-	graphqlHTTP({
-		schema: schema,
+	postgraphile(dbUrl, "public", {
+		ignoreRBAC: true,
+		extendedErrors: ["errcode", "detail", "hint"],
 		graphiql: true,
+		enhanceGraphiql: true,
+		graphqlRoute: "/api",
```
Comment (vuvoth) [inline backend/server.ts:16]: Should we as `/api` and `/graphiql` as constant? I not sure my suggestion is good. How you think about it?

**Review by vuvoth — APPROVED** [review]
I left few comments. 
You should merge it after another member approve PR!

File: backend/server.ts (hunk 3 above)
Comment (robert-zaremba) [reply backend/server.ts:16]: this is used only in one place it's a standard endpoint. I would leave as is.

File: src/api/query.request.ts [hunk 4]
```diff
@@ -1,19 +1,47 @@
 /* GraphQL queries */
-export const getAllDaoReq = (pageNumber: number, pageSize: number) => {
+export const getAllDaoReq = (
+	first: number | null,
+	after: string | null,
+	last: number | null,
+	before: string | null
```
Comment (robert-zaremba) [inline src/api/query.request.ts:14]: please add a doc comment about the parameters.
What's the difference between `first` and  `after`?

**Review by robert-zaremba — APPROVED** [review]
pre-approving

File: src/api/query.request.ts (hunk 4 above)
Comment (rmeena840) [reply src/api/query.request.ts:14]: Done

**Pull request merged**